packages/devtools-reps/src/object-inspector/index.js

```diff
@@ -428,12 +429,24 @@ class ObjectInspector extends Component {
     );
   }
 
+  getDefaultPropsByRep(object) {
```

Comment:
can changes to this file be moved to its own PR ? 
I'm not sure what we are trying to achieve here.

Reply:
Hmm, I was trying to set `omitLinkHref` to false so that links would be rendered. Is there a more proper way to override the default option for the launchpad instead of injecting an options layer only for the purpose of setting `omitLinkHref`, which seems pretty wrong now that I think of it. :|

Reply:
Let's the consumer of the ObjectInspector decude if they want to pass omitLinkHref :)
The default value here is to prevent security issue (a plain link with no way to open the link outside of the devtools toolbox).
The link is always rendered, we only don't add the href attribute